mlir/utils/jenkins/Jenkinsfile

```diff
@@ -712,36 +712,29 @@ pipeline {
                             buildProject('check-rocmlir-build-only ci-performance-scripts', '')
                             dir('build') {
                                 // Tune gemms with default datatypes, fail if the DB is not created
-                                sh """python3 ./bin/tuningRunner.py --operation gemm \
-                                        --configs_file=../mlir/utils/performance/gemm-configs \
-                                        --output=mlir_tuning_${ARCH}.tsv
+                                // (Includes int8xint8->int8 for performance comparisons against CK.)
```

Comment:
In other words, this one tuna-script.sh invocation accounts for both tuningRunner.py invocations at this point.  I chose to have configuration-generation make all the TxT->T cases, and special-cased i8 to make both i8xi8->i8 and i8xi8->i32.